lib/ansible/modules/cloud/vmware/vmware_host_firewall_manager.py

```diff
@@ -44,7 +44,7 @@
     - Each member of list is rule set name and state to be set the rule.
     - Both rule name and rule state are required parameters.
     - Please see examples for more information.
-    default: []
+    default: {}
```

Comment:
rules is a list, not sure if documentation should not it as '{}' instead of '[]' 

@gundalow Could you please suggest something here ?

Reply:
Ok. I fixed it. I needed to remove the 'r'. 

Btw. how is this definition/syntax called?
```
DOCUMENTATION = r'''
[...]
'''
```